src/serviceWorker.ts

```diff
@@ -26,10 +26,10 @@ type Config = {
 };
```

Comment:
I believe this is way of using service workers is dedicated for CRA, we may not need it anymore